### ISSUE TITLE
MaterialDatepickerComponent year selection

### ISSUE THREAD

**Issue by alescdb** [dart-lang/angular_components#350]
Hi everyone,

Unless I'm mistaken, it is not possible to change the year in `MaterialDatepickerComponent`.
So, if you want to select a date far in the past (or future) the only choice is to type it (or mouse scrolling for hours :-)).

It looks like it is possible in `MaterialDateRangePickerComponent` (month by month)

e.g : https://material.angular.io/components/datepicker

![image](https://user-images.githubusercontent.com/5646825/47925331-35a5a200-debe-11e8-8021-c029e6f05ed7.png)

**Comment by cpelling** [comment]
Yes, that's correct. This is possible in MaterialDateRangePickerComponent using the "[Month] [Year]" dropdown, which toggles the date grid to show a month grid, but indeed this is not present in MaterialDatepickerComponent.

We do have a spec for adding an option to include the "[Month] [Year]" dropdown in MaterialDatepickerComponent. It would look pretty similar to the treatment in MaterialDateRangePickerComponent. But nobody's asked for it yet, so it hasn't been implemented. Contributions welcome, as always. :)
